packages/web/config/env.ts

```diff
@@ -18,3 +18,7 @@ export const WALLETCONNECT_RELAY_URL =
 
 export const TIMESERIES_DATA_URL = process.env.NEXT_PUBLIC_TIMESERIES_DATA_URL;
 export const INDEXER_DATA_URL = process.env.NEXT_PUBLIC_INDEXER_DATA_URL;
+
+export const GITHUB_URL = process.env.NEXT_PUBLIC_GITHUB_URL;
```

Comment:
Can we avoid "NEXT_PUBLIC"? If we use this prefix all the env vars will be bundled on the browser, but they're only used on server side, also we have an access token.
https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser

Reply:
As of right now yes. Github APIs are only used server side